lib/svtplay_dl/__init__.py

```diff
@@ -226,11 +226,11 @@ def get_one_media(stream, options):
         return
 
     if options.merge_subtitle:
-        from svtplay_dl.utils import which
-        if not which('ffmpeg'):
-            log.error("--merge-subtitle needs ffmpeg. Please install ffmpeg.")
-            log.info("https://ffmpeg.org/download.html")
-            sys.exit(2)
+        from svtplay_dl.utils import check_ffmpeg
```

Comment:
only `from svtplay_dl.utils import check_ffmpeg`  , move `check_ffmpeg` and put that end of line 13